chroma.go

```diff
@@ -338,7 +344,7 @@ func getMetadatasListFromAPI(metadatas []map[string]openapiclient.MetadatasInner
 			var rawValue interface{}
 			b, e := value.MarshalJSON()
 			if e != nil {
-				rawValue = nil
+				return nil, e
```

Comment:
@tazarov golangci-lint reports a error at this place.  I think we should return this error if not nil